packages/trip-form/src/SettingsSelectorPanel/index.js

```diff
@@ -93,9 +93,13 @@ export default class SettingsSelectorPanel extends Component {
       // If there are multiple (scooter | bikeshare) providers,
       // then if one is specified by the mode button, select it,
       // othewise select all providers.
+      // selectedCompanies is at least an empty array.
       const selectedCompanies =
```

Comment:
Hey Binh ... seeing a console error here:

TypeError: Object(...)(...) is null
2 index.js:97
    SettingsSelectorPanel index.js:97
    handleClick index.js:22
    onClick index.js:32
    React 17
    bind_applyFunctionN self-hosted:1054
    dispatchEvent self-hosted:1018
    unstable_runWithPriority scheduler.development.js:697
    React 4
    dispatchDiscreteEvent self-hosted:977
Source map error: Error: request failed with status 404

SEE IT: click the button on this story:
http://localhost:5555/?path=/story/settingsselectorpanel--settings-selector-panel-undefined-param

Reply:
Thanks, good catch!